CONTRIBUTING.md

```diff
@@ -34,10 +34,9 @@ For building and pushing a new version of the operator image:
  - `make image-build -e IMAGE_TAG_BASE=<image-repo/image-name> VERSION=<semver>`
  - `make image-push -e IMAGE_TAG_BASE=<image-repo/image-name> VERSION=<semver>`
 
-For deploying onto a cluster:
+For running from your host:
  - First, either set `KUBECONFIG` or ensure you are logged into a cluster in your environment
- - `make install`
- - `make deploy -e IMG=<image-repo/image-name>`
+ - `NAMESPACE=default make run`
```

Comment:
why should use "default" ns here, and "make install deploy" becomes "go run"?

Reply:
You are right here.